src/test/java/com/stripe/model/EventDataDeserializerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.stripe.model;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.net.APIResource;
+
+import org.junit.Test;
+
+public class EventDataDeserializerTest extends BaseStripeTest {
+
+  @Test
+  public void testApplicationEventData() throws Exception {
+    final String data = getResourceAsString("/api_fixtures/account_application_deauthorized.json");
+    final Event event = APIResource.GSON.fromJson(data, Event.class);
+    assertNotNull(event);
+    assertNotNull(event.getId());
+
```

Comment:
Can you add a `assertNotNull(event.getData());` and `assertNotNull(event.getData().getObject());` here? That way if either of these is `null` for some reason we'll get a clear assertion error rather than an NPE or cast error in the following line.

Reply:
Sounds good. Done.